drracket/drracket/private/stick-figures.rkt

```diff
@@ -101,67 +101,63 @@
-             [bm-solid (make-object bitmap% w h)]
-             [bm-small (make-object bitmap% w h)]
-             [bdc-big (make-object bitmap-dc% bm-big)]
-             [bdc-solid (make-object bitmap-dc% bm-solid)]
-             [bdc-small (make-object bitmap-dc% bm-small)])
-        (send bdc-big clear)
-        (draw-callback bdc-big small-factor #f points
-                       (+ margin (- (* small-factor min-rx)))
-                       (+ margin #;(- (* small-factor min-ry)))
-                       3)
-        
-        (send bdc-small clear)
-        (send bdc-small set-scale small-bitmap-factor small-bitmap-factor)
-        (send bdc-small draw-bitmap bm-big 0 0)
-        (send bdc-small set-scale 1 1)
-        
-        (send bdc-solid set-brush green 'solid)
-        (send bdc-solid set-pen green 1 'solid)
-        (send bdc-solid draw-rectangle 0 0 w h)
-        
-        (send bdc-solid set-bitmap #f)
-        (send bdc-small set-bitmap #f)
-        (send bdc-big set-bitmap #f)
-        
-        (send bm-solid set-loaded-mask bm-small)
-        bm-solid)))
+  (define-values (min-rx min-ry) (get-max/min-x/y min points))
+  (define-values (max-rx max-ry) (get-max/min-x/y max points))
+  (define margin 2)
+  (define bw (+ margin margin (ceiling (* small-factor (- max-rx min-rx)))))
+  (define bh (+ margin margin (ceiling (* small-factor (- max-ry min-ry)))))
+  (define w (ceiling (* bw small-bitmap-factor)))
+  (define h (ceiling (* bh small-bitmap-factor)))
+  (define bm-big (make-object bitmap% bw bh))
+  (define bm-solid (make-object bitmap% w h))
+  (define bm-small (make-object bitmap% w h))
+  (define bdc-big (make-object bitmap-dc% bm-big))
+  (define bdc-solid (make-object bitmap-dc% bm-solid))
+  (define bdc-small (make-object bitmap-dc% bm-small))
+  (send bdc-big clear)
+  (draw-callback bdc-big small-factor #f points
+                 (+ margin (- (* small-factor min-rx)))
+                 (+ margin #;(- (* small-factor min-ry)))
+                 3)
+  (send bdc-small clear)
+  (send bdc-small set-scale small-bitmap-factor small-bitmap-factor)
+  (send bdc-small draw-bitmap bm-big 0 0)
+  (send bdc-small set-scale 1 1)
+  (send bdc-solid set-brush green 'solid)
+  (send bdc-solid set-pen green 1 'solid)
+  (send bdc-solid draw-rectangle 0 0 w h)
+  (send bdc-solid set-bitmap #f)
+  (send bdc-small set-bitmap #f)
+  (send bdc-big set-bitmap #f)
+  (send bm-solid set-loaded-mask bm-small)
+  bm-solid)
   
   (define (get-running-bitmap) (get-bitmap running-points (make-object color% 30 100 30)))
   (define (get-waiting-bitmap) (get-bitmap waiting-points (make-object color% 30 100 30)))
   
   (define (normalize points)
-    (let-values ([(min-x min-y) (get-max/min-x/y min points)])
-      (map (λ (x) (list (car x) 
-                        (+ (- (list-ref x 1) min-x))
-                        (+ (- (list-ref x 2) min-y))))
-           points)))
+  (define-values (min-x min-y) (get-max/min-x/y min points))
```

Comment:
This line also doesn't look indented inside `normalize`

Reply:
It's on the same line as `define-values`

Reply:
Wait I see what you mean now. Let me take a closer look.

Reply:
Turns out this entire file is incorrectly indented. Resyntax is fixing the indentation for just this code, but because it doesn't fix it for the lines before the change, it looks off.